### DISCUSSION TITLE
wire:model.change on checkbox does not contain all values

### DISCUSSION THREAD

**Discussion by AyKay35** [livewire/livewire#7581] [7. Bugs (archived)]
### Livewire version

v3.3.0

### Laravel version

v.10.35.0

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

1. Click on a checkbox and you will receive a DD
2. Click on the second checkbox, but you wont receive both values

Since I want to use this in my project as filter, I really need to react on changes thats why i use wire:model.change.

### Repository or Wirebox URL

https://wirebox.app/b/4od9e

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by AyKay35** [comment]
does no one have an idea or clue? Am I using this wrong or is it a bug?

**Reply by kirkbushell** [reply]
Yup, this has been noted in a separate issue - it's clear there's a problem here with checkbox arrays.

**Comment by snapey** [comment]
The problem here is the use of dd which prevents the update running for a second time to update the next checkbox.
